test/python/api_config.py

```diff
@@ -18,6 +19,9 @@
 WEBSERVICE_POLICY = pathlib.Path('test/config/webservice.yml')
 OIDC_POLICY_FILE = 'test/config/oidc-webservice.yml'
 
+if (DAP_TESTS := os.environ.get('DAP_TESTS', default=False)) == '1':
+    DAP_TESTS = True
```

Comment:
```suggestion
if (ENTERPRISE_TESTS := os.environ.get('ENTERPRISE_TESTS', default=False)) == '1':
    ENTERPRISE_TESTS = True
```
let's get rid of all the DAPs now